### PR TITLE
Read driver config from configuration.toml by sdk API

### LINKED ISSUE #34: Read driver config from configuration.toml by sdk API
According to PR https://github.com/edgexfoundry/device-sdk-go/pull/207  is merged, device-mqtt can add config in configration.tmol and read by device-sdk-go API.

### PR DESCRIPTION
**Pull request by weichou1229**
Fix #34

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -55,3 +55,24 @@ LogLevel = "INFO"
...
+IncomingSchema = "tcp"
+IncomingHost = "0.0.0.0"
+IncomingPort = "1883"
+IncomingUser = "admin"
+IncomingPassword = "public"
+IncomingQos = "0"
+IncomingKeepAlive = "3600"
+IncomingClientId = "IncomingDataSubscriber"
+IncomingTopic = "DataTopic"
+
```
Comment (cloudxxx8) [inline cmd/res/configuration.toml]: why empty line here?

File: cmd/res/configuration.toml [hunk 2]
```diff
@@ -55,3 +55,24 @@ LogLevel = "INFO"
...
+ResponseSchema = "tcp"
+ResponseHost = "0.0.0.0"
+ResponsePort = "1883"
+ResponseUser = "admin"
+ResponsePassword = "public"
+ResponseQos = "0"
+ResponseKeepAlive = "3600"
+ResponseClientId = "CommandResponseSubscriber"
+ResponseTopic = "ResponseTopic"
```
Comment (cloudxxx8) [inline cmd/res/configuration.toml]: add empty line

File: cmd/res/example/configuration.toml [hunk 3]
```diff
@@ -68,3 +68,24 @@ Level = "DEBUG"
...
+IncomingSchema = "tcp"
+IncomingHost = "0.0.0.0"
+IncomingPort = "1883"
+IncomingUser = "admin"
+IncomingPassword = "public"
+IncomingQos = "0"
+IncomingKeepAlive = "3600"
+IncomingClientId = "IncomingDataSubscriber"
+IncomingTopic = "DataTopic"
+
```
Comment (cloudxxx8) [inline cmd/res/example/configuration.toml]: why empty line?

Comment (weichou1229) [reply cmd/res/example/configuration.toml]: I want to separate  Incoming and Response settings

File: cmd/res/configuration.toml (hunk 2 above)
Comment (weichou1229) [reply cmd/res/configuration.toml]: Added.

**Pull request merged**